yourbench/pipeline/single_shot_question_generation.py

```diff
@@ -291,19 +291,6 @@ def _process_responses_and_build_dataset(
                 logger.warning(
                     f"No parseable JSON found (or empty list) for row_index={row_index}, chunk_id={chunk_id}, model={model_name}. Creating fallback row."
```

Comment:
```suggestion
                    f"No parseable JSON found (or empty list) for row_index={row_index}, chunk_id={chunk_id}, doc_id={doc_id}, and model={model_name}."

Reply:
unfortunately, we’ll also need to cascade catch this during answer generation, judging, etc, which seems like asking for inaccuracies / errors.

+1 for logging it somewhere however

Reply:
Yep I agree - just want to make sure with Alina that there isn't something we haven't thought of handled here

Reply:
yes I did. the only difference is that we no longer have an "unknown" "unknown" row that would cause cascading errors